app/prometheus/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+/bin/sed -i 's/{GRAFANA_CLOUD_TOKEN}/'$GRAFANA_CLOUD_TOKEN'/' /etc/prometheus/prometheus.yml
+/bin/sed -i 's/{PROMEHTEUS_ENVIRONMENT}/'$PROMEHTEUS_ENVIRONMENT'/' /etc/prometheus/prometheus.yml
+/bin/prometheus --config.file=/etc/prometheus/prometheus.yml \
+             --storage.tsdb.path=/prometheus \
```

Comment:
So just to check: it doesn't matter if prometheus dies and this tsdb gets nuked? The actual data is pushed to grafana(?)

Reply:
That is correct. The full path is: application hosts metrics on port 8000/8001 -> scraped by our prometheus -> our prometheus forwards them to the prometheus instance hosted by the folks at Grafana labs -> the grafana application in our Grafana labs setup shows graphs from the Grafana labs central prometheus.